packages/flutter/lib/src/widgets/async.dart

```diff
@@ -372,42 +403,87 @@ typedef AsyncWidgetBuilder<T> = Widget Function(BuildContext context, AsyncSnaps
 // TODO(ianh): remove unreachable code above once https://github.com/dart-lang/linter/issues/1139 is fixed
 class StreamBuilder<T> extends StreamBuilderBase<T, AsyncSnapshot<T>> {
   /// Creates a new [StreamBuilder] that builds itself based on the latest
-  /// snapshot of interaction with the specified [stream] and whose build
+  /// snapshot of interaction with the specified `stream` and whose build
   /// strategy is given by [builder].
   ///
-  /// The [initialData] is used to create the initial snapshot.
+  /// The [initialData] argument is used to create the initial snapshot.
   ///
   /// The [builder] must not be null.
```

Comment:
docs should talk about what to do if you don't have the initial data (the common case)